endgame_postprocessing/post_processing/iu_data.py

```diff
@@ -39,11 +41,11 @@ class IUSelectionCriteria(Enum):
 class IUData:
 
     def __init__(
-        self,
-        input_data: pd.DataFrame,
-        disease: Disease,
-        iu_selection_criteria: IUSelectionCriteria,
-        simulated_IUs: set[str] = None,
+            self,
```

Comment:
❓ We should find out what formatting is being applied automatically

Reply:
Let's agree to use Black - but when I format this document with black these do not get indented.